backend/Controllers/CardController.cs

```diff
@@ -17,6 +18,7 @@ public CardController(DatabaseContext dbContext)
             _dbContext = dbContext;
         }
 
+        [Authorize(Policy = "Admin")]
         [HttpGet]
         public async Task<ActionResult<List<Card>>> GetCards()
         {
```

Comment:
When testing on Android, the only problem is that currently, Android application performs `GET` call for `GetCards` whenever checking if RFID/BLE card authentication is valid in order to get all card addresses and perform comparison for match on Android itself, but `GetCards` should be reserved for admins only, since it returns all card addresses from database and Android application shouldn't have access to them from security standpoint, since the card addresses can be sniffed if they are sent over network. Currently, with admin authorization in place for this method, Android card authentication won't work. In order to ensure `GetCards` keeps this admin-only authorization, another `GET` method which doesn't require `[Authorize]` attribute (`AuthenticateCardAddress` or similar) should be added to `CardController`. It can receive a card address string, perform comparison with all database card addresses and return true if a match is found, false otherwise. That way, Android application can call this method instead of `GetCards` which is now inaccessible to it.

Reply:
I made endpoint for that